apiserver/pkg/manager/resource_manager.go

```diff
@@ -360,7 +362,8 @@ func (r *ResourceManager) ListAllServices(ctx context.Context) ([]*rayv1api.RayS
 	}
 
 	for _, namespace := range namespaces.Items {
-		servicesByNamespace, err := r.ListServices(ctx, namespace.Name)
+		// TODO(tinaxfwu): Implement list all services with pagination support.
```

Comment:
```suggestion
```
Is the name in the `()` correct? Since we already track issues on github, I think we don't need the TODO here.

Reply:
It's intentional, because Tina takes the left work; but I could remove it. :)